Bump Cruise Control to v2.5.37 for Kafka 2.7 support (#4487)
strimzi/strimzi-kafka-operator: CHANGELOG.md

* Add support for Ingress v1 in Kubernetes 1.19 and newer
* Add support for Kafka 2.6.1
* List topics used by a Kafka Connect connector in the `.status` section of the `KafkaConnector` custom resource

> * Bump Cruise Control to v2.5.37 for Kafka 2.7 support. Note this new version of Cruise Control uses `log4j2` and is supported by dynamic logging configuration. However, existing `log4j` configurations must be updated to `log4j2` configurations.

Reviewer: ```suggestion
* Bump Cruise Control to v2.5.37 for Kafka 2.7 support. Note this new version of Cruise Control uses `log4j2` and is supported by dynamic logging configuration (where logging properties are defined in a ConfigMap). However, existing `log4j` configurations must be updated to `log4j2` configurations.
```

> * Bump Cruise Control to v2.5.37 for Kafka 2.7 support. Note this new version of Cruise Control uses `log4j2` and is supported by dynamic logging configuration. However, existing `log4j` configurations must be updated to `log4j2` configurations.

Reviewer 2: There is also another `Updated Cruise Control to version 2.5.32` message above which should be removed.

* Add support for Ingress v1 in Kubernetes 1.19 and newer
* Add support for Kafka 2.6.1
* List topics used by a Kafka Connect connector in the `.status` section of the `KafkaConnector` custom resource
* Bump Cruise Control to v2.5.37 for Kafka 2.7 support.

> * New Cruise Control supports `log4j2` implementation thus we support dynamic logging changes for it. The logging configuration has to be changed.

Reviewer 2: Maybe this should be in single bullet point?

        name: my-config-map
        key: my-key
  ``` 

> * Exisiting Cruise Control logging configurations must be updated from `log4j` syntax to `log4j2` syntax.

Reviewer: ```suggestion
* Existing Cruise Control logging configurations must be updated from `Log4j` syntax to `Log4 j2` syntax.
```

  * For existing inline configurations, replace the `cruisecontrol.root.logger` property with `rootLogger.level`.

>   * For existing external configurations, replace the existing configuration a new configuration file named `log4j2.properties` using `log4j2` syntax. 

Reviewer: ```suggestion
  * For existing external configurations, replace the existing configuration with a new configuration file named `log4j2.properties` using `log4j2` syntax. 
```

        name: my-config-map
        key: my-key
  ``` 

> * Cruise Control logging configuration does not contain `cruisecontrol.root.logger` property anymore. It has been replaced by `rootLogger.level`.

Reviewer 3: So will this mean anyone who was running 0.21.x CC with their own logging config will need to change that when they upgrade? Or will the changes in this PR do that automatically?

Author: Yes, from what I understand users will need to update their logging configurations to use `log4j2` syntax when they upgrade from 0.21.x CC.

> * Cruise Control logging configuration does not contain `cruisecontrol.root.logger` property anymore. It has been replaced by `rootLogger.level`.

Reviewer 2: I do not think this is enough. I assume that users with inline logging need to change this. But additionally, any users with external logging need to have completely new logging file and have it stored under a new `log4j2.properties` file.

Author: I adding some more text here, let me know what you think!
